src/engine/math_util.c

```diff
@@ -151,21 +149,19 @@ void vec3f_copy_y_off(Vec3f dest, Vec3f src, f32 yOff) { vec3_copy_y_off_func(f3
 #undef vec3_copy_y_off_func
 
 /// Set vector 'dest' to (x, y, z)
-inline void vec3f_set(Vec3f dest, const f32 x, const f32 y, const f32 z) { vec3_set(dest, x, y, z); }
-inline void vec3i_set(Vec3i dest, const s32 x, const s32 y, const s32 z) { vec3_set(dest, x, y, z); }
-inline void vec3s_set(Vec3s dest, const s16 x, const s16 y, const s16 z) { vec3_set(dest, x, y, z); }
+ALWAYS_INLINE void vec3f_set(Vec3f dest, const f32 x, const f32 y, const f32 z) { vec3_set(dest, x, y, z); }
+ALWAYS_INLINE void vec3i_set(Vec3i dest, const s32 x, const s32 y, const s32 z) { vec3_set(dest, x, y, z); }
+ALWAYS_INLINE void vec3s_set(Vec3s dest, const s16 x, const s16 y, const s16 z) { vec3_set(dest, x, y, z); }
 
```

Comment:
heres a screencap cus i'm too lazy to copy and paste

![image](https://user-images.githubusercontent.com/79979276/145837132-e6ba14c6-f5f1-4fc9-ae46-df09a41ae1a7.png)